### PR TITLE
feat(modal): show dialog text

### PR DESCRIPTION
**Pull request by yoshuawuyts**
Implements the final modal features. Ref #103

### CODE REVIEW

File: elements/modal.js [hunk 1]
```diff
@@ -121,15 +122,17 @@ const prefix = css`
 module.exports = createWidget
 
 function createWidget () {
+  let isCopied = false
```
Comment (juliangruber) [inline elements/modal.js]: i wonder if this is a little over engineered, couldn't the flow also just be `open modal -> not copied. copy -> copied. close modal -> reset state`

Comment (yoshuawuyts) [inline elements/modal.js]: That's what we're doing right? Or are you talking about the unidirectional rendering stuff being too complicated? - I think I definitely prefer explicit state and re-rendering over using a selector and some sort of "toggle class" method.

Would perhaps adding a method called `reset` and calling that when we reset stuff help?

**Comment by yoshuawuyts** [comment]
Did a round of updates incorporating @juliangruber's feedback - code should be wayyy simpler now :D

**Comment by yoshuawuyts** [comment]
@juliangruber thanks for the quick review!

**Pull request merged**